player-app/src/main/java/org/gameontext/player/AllPlayersResource.java

```diff
@@ -180,11 +180,19 @@ public Response createPlayer(PlayerArgument player) throws IOException {
 
         PlayerDbRecord pFull = new PlayerDbRecord();
         pFull.update(player);   // get all proposed updates
+        
         Claims claims = (Claims) httpRequest.getAttribute("player.claims");
 
         // make sure an API key is generated for the new user
         pFull.setApiKey(SharedSecretGenerator.generateApiKey());
-
+        
+        //add in story/playerMode if they are present in the users auth jwt.
+        if(claims.containsKey("story")) {
+          pFull.setStory(claims.get("story", String.class));
```

Comment:
PLEASE make your vim pay attention to .editorconfig files. :-P
Or eclipse
or vscode
or..